lambdas/smartblocks-store_get.ts

```diff
@@ -31,10 +24,11 @@ const getWorkflow = ({
           TableName: "RoamJSSmartBlocks",
           Item: {
             uuid: { S: nanoid() },
-            name: { S: item.uuid.S },
+            name: { S: item?.uuid.S },
             author: { S: graph },
-            workflow: { S: item.workflow.S },
+            workflow: { S: item?.workflow.S },
             status: { S: toStatus("INSTALLED") },
+            installed: { S: new Date().toJSON() }, // let's see if anyone is actually using this
```

Comment:
this is the only notable change, the rest is typescript cleanup.

I'd like to replace the smartblocks store with SamePage, so I'm adding this field here to track how much this is even getting used until then